cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -36,6 +36,8 @@ def cleanup_vm(vm_name, provider_key, provider_mgmt):
 def pytest_generate_tests(metafunc):
     # Filter out providers without provisioning data or hosts defined
     argnames, argvalues, idlist = testgen.infra_providers(metafunc, 'provisioning')
+    if metafunc.function.func_name == 'test_disk_format_select':
```

Comment:
Another thought, along these lines: Instead of checking the test name, would it make sense to move the tests that need to be parametrized in this way into their own module?